src/main/java/com/checkout/disputes/DisputesQueryFilter.java

```diff
@@ -1,26 +1,62 @@
 package com.checkout.disputes;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.Builder;
 import lombok.Data;
 
 import javax.validation.constraints.Size;
+import java.time.Instant;
 
 @Data
 @Builder
-public final class DisputesQueryFilter {
+public class DisputesQueryFilter {
 
     @Size(min = 1, max = 250)
     private Integer limit;
+
     @Size
     private Integer skip;
-    private String from;
-    private String to;
+
+    private Instant from;
+
+    private Instant to;
+
     private String id;
-    /** One or more comma-separated statuses. This works like a logical OR operator */
+
+    /**
+     * One or more comma-separated statuses. This works like a logical OR operator
+     */
     private String statuses;
+
+    @SerializedName("payment_id")
     private String paymentId;
+
+    @SerializedName("payment_reference")
     private String paymentReference;
+
+    @SerializedName("payment_arn")
     private String paymentArn;
-    private String thisChannelOnly;
+
+    @SerializedName("this_channel_only")
+    private boolean thisChannelOnly;
+
+    /**
+     * Available only on Four
+     */
+
+    /**
+     * One or more comma-separated client entities. This works like a logical OR operator
+     */
+    @SerializedName("entity_ids")
+    private String entityIds;
```

Comment:
Can we make this a `List` and then join them with "," so users don't have to do it? Same for `subEntityIds`.